source/crud/tests/unified/findOne.yml

```diff
@@ -0,0 +1,60 @@
+description: "findOne"
```

Comment:
These tests heavily borrow from the `find` tests. I've decided to only explicitly check for `limit` and the absence of `batchSize` in the outgoing command. I don't think there are any other special cases we need to test.